pkg/gui/diff_context_size_test.go

```diff
@@ -0,0 +1,41 @@
+package gui
+
+import (
+	"testing"
+
+	"github.com/jesseduffield/gocui"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestIncreasesContextInDiffViewByOne(t *testing.T) {
```

Comment:
I appreciate the extensive testing!

Reply:
I'm a TDD head, I was very glad to see that you already had testing infrastructure in place :smile: 
This isn't always the case, both in open- and closed source projects.